src/pages/ui/settings.page.ts

```diff
@@ -0,0 +1,211 @@
+
+  async openProfilNewPasswordField() {
+    await this.profilNewPassword.click();
+  }
+
+  async openProfilNewPasswordInput(newPassword: string) {
+    await this.profilNewPassword.fill(newPassword);
+  }
+
+  async openProfilNewPasswordAgainField() {
+    await this.profilNewPasswordAgain.click();
+  }
+
+  async openProfilNewPasswordAgainInput(newPasswordAgain: string) {
+    await this.profilNewPasswordAgain.fill(newPasswordAgain);
+  }
+
+  async openProfilChangePasswordB() {
+    await this.profilChangePasswordB.click();
+  }
+
+  async openAccessCodeAddSaveDelete(accessCode: string): Promise<SettingsPage> {
+    await this.openProfil();
+    await this.openProfilMAC();
+    await this.openProfilAAC();
+    await this.openProfilACField();
+    await this.accessCodeInput(process.env.HOBBYFARM_ACCESS_CODE as string);
+    await this.openProfilSAAC();
+    await expect(this.page.getByRole('gridcell', { name: process.env.HOBBYFARM_ACCESS_CODE })).toHaveText(process.env.HOBBYFARM_ACCESS_CODE as string);
+    await this.openSelectAC();
+    await this.openDelete();
+    await this.openClose();
+    return new SettingsPage(this.page);
+  }
+
+  async openAccessCodeUnique(): Promise<SettingsPage> {
+    await this.openProfil();
+    await this.openProfilMAC();
+    await this.openProfilAAC();
+    await this.openProfilACField();
+    await this.accessCodeInput(process.env.HOBBYFARM_ACCESS_CODE as string);
+    await this.openProfilSAAC();
+    await this.openProfilAAC();
+    await this.openProfilACField();
+    await this.accessCodeInput(process.env.HOBBYFARM_ACCESS_CODE as string);
+    await this.openProfilSAAC();
+    await this.openErrorAC();
+    await this.openSelectAC();
+    await this.openDelete();
+    await this.openClose();
+    return new SettingsPage(this.page);
+  }
+
+  async openSettingsChangeFontSize(): Promise<SettingsPage> {
+    await this.openProfil();
+    await this.openProfilSettings();
+    await this.openFontSizeField();
+    await this.openProfilFontSizeUp();
+    await this.openProfilSave();
+    await this.openProfil();
+    await this.openProfilSettings();
+    await this.openFontSizeField();
+    await this.openProfilFontSizeDown();
+    await this.openProfilSave();
+    return new SettingsPage(this.page);
+  }
+
+  async openPasswordChangeTest(): Promise<SettingsPage> {
+    await this.openProfil();
+    await this.openProfilChangePassword();
+    await this.openProfilOldPasswordField();
+    await this.openProfilOldPasswordInput(process.env.HOBBYFARM_ADMIN_UI_PWD as string);
+    await this.openProfilNewPasswordField();
+    await this.openProfilNewPasswordInput(process.env.HOBBYFARM_ADMIN_UI_PWD_TEST as string);
+    await this.openProfilNewPasswordAgainField();
+    await this.openProfilNewPasswordAgainInput(process.env.HOBBYFARM_ADMIN_UI_PWD_TEST as string);
+    await this.openProfilChangePasswordB();
+    return new SettingsPage(this.page);
+  }
+
+  async openPasswordChangeBack(): Promise<SettingsPage> {
```

Comment:
It would be best to not use process.env here and pass it as parameters instead and the function body should be indented